1-Draft/RFCxxxx-PSModulePath.md

```diff
@@ -0,0 +1,159 @@
+---
+RFC:
+Author: Steve Lee
+Status: Draft
+SupercededBy: n/a
+Version: 1.0
+Area: Engine
+Comments Due: 11/30
+Plan to implement: Yes
+---
+
+# PSModulePath When Starting PowerShell within PowerShell
+
+When starting PowerShell from a different version of PowerShell, the PSModulePath
+should reflect the module search path and only include segments appropriate for
+the version of PowerShell being started.
+
+## Motivation
+
+    As a PowerShell User,
+    I can start a different version of PowerShell within PowerShell,
+    so that I can do things required by that specific version of PowerShell.
+
+## User Experience
+
+### Starting Windows PowerShell from PowerShell 7
+
+    ```powershell
```

Comment:
Unindent these code blocks. The MD code fences are being treated as code because of the indentation. This causes a rendering problem